utils/src/grpc.rs

```diff
@@ -13,7 +13,15 @@ use std::{
     },
     task::{Context, Poll},
 };
-use tower_http::map_request_body::MapRequestBodyLayer;
+pub use tower::ServiceBuilder;
+pub use tower_http::map_request_body::MapRequestBodyLayer;
+pub use tower_http::map_response_body::MapResponseBodyLayer;
+
+#[derive(Default, Clone, Debug)]
+pub struct GrpcCounters {
```

Comment:
I want to reduce the footprint of this as much as possible right now, which is why I scoped this strictly under a "gRPC utility" module. Is there a common base that can house this tiny struct that is accessible by p2p flows and gRPC client?  Technically this can be split into `kaspa_utils::hyper` and `kaspa_utils::grpc` but it looks fine to me the way it is.  I'd like this to have as a reduced footprint as possible and am currently treating this as auxiliary utilities serving the gRPC layer.

Reply:
Ok, I split it into `kaspa_utils::hyper` and `kaspa_utils::grpc`.